kolibri/logger/models.py

```diff
@@ -271,5 +271,5 @@ class ExamAttemptLog(BaseAttemptLog):
     content_id = UUIDField()
     channel_id = UUIDField()
 
-    def infer_dataset(self):
+    def infer_dataset(self, *args, **kwargs):
```

Comment:
It's unclear why all these methods just receive and throw away all their arguments!?